### DISCUSSION TITLE
Disable Alt key entering "focus to pointer"

### DISCUSSION THREAD

**Discussion by lgander** [pubpub-zz/ppInk#272] [Q&A] [answered]
First, let me thank you for this amazing software. I use it a lot.

There is a function that I can't disable. I can't even find it in the documentation nor in the hotkey section of the options:
After ppink is enabled, when I press Alt there is something similar as a "focus around the cursor position".
Is there any way to disble this? Or change the hotkey?

Thanks again!

**Comment by pubpub-zz** [comment, marked as answer]
The option can be disabled in the options:
![image](https://github.com/pubpub-zz/ppInk/assets/4083478/cdf548de-930f-4d06-be3e-57756552d7da)
in the config.ini file  it is : `Spot_On_Alt`

**Reply by lgander** [reply]
Cant believe how I missed it. Thanks a lot

**Comment by pubpub-zz** [comment]
Do not hesitate to star the project 🙂

**Reply by lgander** [reply]
Done :)
